### PR TITLE
Add support for ppt,pttx,js,hta extensions

### PR DESCRIPTION
**Pull request by kscieslinski**
**Note**: I've renamed the `office.py` back to `vba_graph`. After `hid emulation` will be added to drakvuf we can get rid of it.

### CODE REVIEW

File: drakrun/drakrun/sample_startup.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import subprocess
+from oletools.olevba import VBA_Parser
+
+from drakrun.vba_graph import get_outer_nodes_from_vba_file
+
+
+log = logging.getLogger("drakrun")
+
+
+class SampleStartupRouter:
```
Comment (chivay) [inline drakrun/drakrun/sample_startup.py:13]: Is there any particular reason to add this class?
IMO we could just put the methods in the module as plain functions.

Comment (kscieslinski) [reply drakrun/drakrun/sample_startup.py:13]: There is no, it was just cleaner for me

Comment (chivay) [reply drakrun/drakrun/sample_startup.py:13]: I mean, `sample_startup` already works as a namespace for those functions. :thinking: 
We can import it with:
```
import drakrun.sample_startup as sample_startup
```
and use with `sample_startup.get_sample_startup_command`

Comment (kscieslinski) [reply drakrun/drakrun/sample_startup.py:13]: fine with me, I will change this

**Review by chivay — APPROVED** [review]

**Pull request merged**